### ISSUE TITLE
chef-client 13 error uninitialized constant Chef::Resource::WsusServerConfiguration

### ISSUE THREAD

**Issue by Stromweld** [criteo-cookbooks/wsus-server#25]
Getting this error below when running on windows 2012 r2 and chef-client 13.1.31. Previous chef-client 12 did work. 

````
Recipe: wsus-server::configure
  * wsus_server_configuration[Wsus server configuration] action configure

    ================================================================================
    Error executing action `configure` on resource 'wsus_server_configuration[Wsus server configuration]'
    ================================================================================

    NameError
    ---------
    uninitialized constant Chef::Resource::WsusServerConfiguration

    Resource Declaration:
    ---------------------
    # In C:/chef/cache/cookbooks/wsus-server/recipes/configure.rb

     27: wsus_server_configuration 'Wsus server configuration' do
     28:   master_server                                    server_conf['master_server']
     29:   update_languages                                 server_conf['update_languages']
     30:   properties                                       server_conf['properties']
     31: end
     32:

    Compiled Resource:
    ------------------
    # Declared in C:/chef/cache/cookbooks/wsus-server/recipes/configure.rb:27:in `from_file'

    wsus_server_configuration("Wsus server configuration") do
      action :configure
      default_guard_interpreter :default
      properties {"TargetingMode"=>"Client", "UpstreamWsusServerName"=>"pshovm-app01.teamfreeze.com", "SyncFromMicrosoftUpdate"=>false, "IsReplicaServer"=>true, "UpstreamWsusServerPortNumber"=>8530, "UpstreamWsusServerUseSsl"=>false}
      declared_type :wsus_server_configuration
      cookbook_name "wsus-server"
      recipe_name "configure"
      update_languages ["en"]
    end

    System Info:
    ------------
    chef_version=13.1.31
    platform=windows
    platform_version=6.3.9600
    ruby=ruby 2.4.1p111 (2017-03-22 revision 58053) [x64-mingw32]
    program_name=C:/opscode/chef/bin/chef-client
    executable=C:/opscode/chef/bin/chef-client


Running handlers:
[2017-06-05T09:23:30-05:00] ERROR: Running exception handlers
Running handlers complete
[2017-06-05T09:23:30-05:00] ERROR: Exception handlers complete
Chef Client failed. 5 resources updated in 01 minutes 18 seconds
[2017-06-05T09:23:31-05:00] FATAL: Stacktrace dumped to C:/chef/cache/chef-stacktrace.out
[2017-06-05T09:23:31-05:00] FATAL: Please provide the contents of the stacktrace.out file if you file a bug report
[2017-06-05T09:23:31-05:00] FATAL: NameError: wsus_server_configuration[Wsus server configuration] (wsus-server::configure line 27) had an error: NameError: uninitialized constant Chef::Resource::WsusServerConfiguration
````

**Comment by Annih** [comment]
Hello @Stromweld 

Thanks for this report.
Wsus-server currently does not support chef 13.

We'll have a look ASAP.

**Comment by Annih** [comment]
Fixed by #29
